src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -157,6 +157,15 @@ struct SvgLineNode
     float y2;
 };
 
+struct SvgImageNode
+{
+    float x;
```

Comment:
One line please.
float x, y, w, h;

Reply:
one line there would be odd. File contain multiple structs definitions in the same manner.

Reply:
@mmaciola I'd like to guide you a convention. See  other files. ex) tvgSwCommon.h